pages/linux/btrfs-property.md

```diff
@@ -0,0 +1,24 @@
+# btrfs property
+
+> Get/Set/List properties for given btrfs filesystem object (file/directory/subvolume/filesystem/device).
```

Comment:
Perhaps this will read better?
```suggestion
> Get, set, or list properties for a given btrfs filesystem object (files, directories, subvolumes, filesystems, or devices).
```